Cargo.toml

```diff
@@ -87,6 +87,11 @@ getrandom = { version = "0.3", default-features = false }
 gloo-timers = "0.3"
 hex = { package = "const-hex", version = "1.14" }
 hkdf = "0.12.3"
+hpke-rs-xmtp = { version = "0.2.1-alpha.1", git = "https://github.com/cryspen/hpke-rs", tag = "v0.2.1-alpha.1", package = "hpke-rs", features = [
+  "hazmat",
+  "serialization",
+  "libcrux",
+] }
```

Comment:
This is the version that is currently relied on. I will try just upgrading to the available 0.3.0 release from crates.io.